cluster/manifests/wiz/sensor/dameonset.yaml

```diff
@@ -0,0 +1,220 @@
+            - DAC_OVERRIDE # file hashing
+            - DAC_READ_SEARCH # file hashing
+            - NET_ADMIN # network events
+            - NET_RAW # network events
+            - IPC_LOCK # eBPF
+            - FOWNER # file hashing
+            - SYS_PTRACE # eBPF
+            - SYSLOG # kernel symbol resolve
+            - KILL # response
+          privileged: false
+          runAsNonRoot: true
+          runAsUser: 2202
+          runAsGroup: 2202
+          readOnlyRootFilesystem: true
+          seccompProfile:
+            type: Unconfined
+          seLinuxOptions:
+            user: "system_u"
+            role: "system_r"
+            level: "s0"
+            type: "spc_t"
+
+        env:
+        - name: MIN_SENSOR_COMPAT_BUILD_VER
+          value: "4142"
+        - name: HELM_CHART_COMPAT_VER
+          value: "1"
+        - name: WIZ_TMP_STORE
+          value: "/wiz-sensor-store/"
+        - name: WIZ_HOST_STORE
+          value: "/wiz-host-cache/"
+        - name: WIZ_RAMFS_STORE
+          value: "/tmp/"
+        - name: LOG_FILE
+          value: "/wiz-sensor-store/sensor.log"
+        - name: MY_NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        - name: MY_POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: MY_POD_UID
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.uid
+        - name: MY_POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: MY_APIKEY_SECRET_NAME
+          value: wiz-sensor-apikey
+        - name: AWS_EC2_METADATA_DISABLED
+          value: "true"
+        - name: CRI_SOCKET_CUSTOM_PATH
+          value: 
+        - name: HTTP_PROXY_URL
+          value: 
+        - name: HTTP_PROXY_USERNAME
+          value: 
+        - name: HTTP_PROXY_PASSWORD
+          value: 
+        - name: HTTP_PROXY_CERT
+          value: 
+        - name: BACKEND_ENV
+          value: prod
+        - name: SENSOR_CLUSTER_NAME
+          value: 
+        - name: RUST_LOG
+          value: info
+        - name: STDOUT_LOG
+          value: error
+        - name: POD_MEM_LIMITS
+          valueFrom:
+            resourceFieldRef:
+              containerName: wiz-sensor
+              resource: limits.memory
+              divisor: "1Mi"
+        - name: POD_CPU_LIMITS
+          valueFrom:
+            resourceFieldRef:
+              containerName: wiz-sensor
+              resource: limits.cpu
+              divisor: "1m"
+
+        volumeMounts:
+        - name: sensor-host-cache
+          mountPath: /wiz-host-cache/
+        - name: sensor-store
+          mountPath: /wiz-sensor-store/
+        - name: tmp-store
+          mountPath: /tmp/
+        - name: api-client-secret
+          mountPath: /api-client/
+        - name: api-endpoint-name-secret
+          mountPath: /api-endpoint-name/
+        resources:
```

Comment:
Perhaps we should add config items to control the cpu/mem resources, this would also make it easier to adjust in the future per cluster or as an emergency response.

See an example here:
https://github.com/zalando-incubator/kubernetes-on-aws/blob/3d593d0065e198c05ee5c26ac720e644bd52e853/cluster/manifests/audittrail-adapter/daemonset.yaml#L71-L77

Reply:
+1 to this suggestion. We have seen time and again the usefulness of setting values as config items.

Reply:
as we are updating the resources at globally here https://github.bus.zalan.do/teapot/cluster-config-manager/blob/cbdb347320c35c0b6e6860479f243a4c36aba064/clusters/secops-test.yaml#L5 , we can keep the resources constant across all clusters

Reply:
@vinaythupili the reserved value is to indicate how much space is free for user pods on a node and to protect us from disrupting their workloads, however it doesn't control the actual resources used by a specific Daemonset.

Reply:
updated